src/components-css/Dropdown.css

```diff
@@ -0,0 +1,46 @@
+#dropdownContainer {
+  display: flex;
+  flex-direction: column;
+}
+
+label {
+  font-weight: bold;
+  font-size: 16px;
+}
```

Comment:
You should specify which labels (and other things like labels) you are targeting (like in which divs or sections they belong to), instead of just using the label tag simply on its own, because this can also cause other labels in other components of yours to be affected by this styling (I know because this happened to me and then I couldn't figure out what was wrong with my styling, and it ended up being because of styling I had in another component). Unless of course you want all labels to have the same styling across your whole app, in which case stuff like this should probably go in your App.css